tools/smoke_test.py

```diff
@@ -0,0 +1,166 @@
+import os
+import re
+import subprocess
+import sys
+import traceback
+
+try:
+    import packaging.version
+except ImportError:
+    from pkg_resources import packaging
+Version = packaging.version.Version
+
+MIN_CUDA_VERSION = Version("11.6")
+MIN_PYTHON_VERSION = (3, 7)
+# NOTE: dev version required, e.g. Version('1.13.0.dev123') < Version('1.13.0')
+MIN_TORCH_VERSION = Version("1.13.0.dev0")
```

Comment:
Is that correct?

Reply:
Not sure what the concern is here?

Reply:
I was under the impression that with all the fake tensor stuff, we basically are only able to run on nightly torch. Just want to make sure this min torch version is correct.

Reply:
https://peps.python.org/pep-0440/#summary-of-permitted-suffixes-and-relative-ordering, `dev0` is the smallest in the version sorting order, and we aren't using a version of torch that looks like `1.13aN`.